teamA/jeongHaeMan/.env.sample

```
@@ -0,0 +1,9 @@
+PORT=3000
+
```

dbmate 설정용 DATABASE_URL도 초기세팅용 환경변수 샘플파일내에 추가해주세요!

```suggestion
DATABASE_URL="database_client://username:password@localhost:port_number/database_name"
PORT=3000
```